packages/language-server/test/snapshots/xml-view-diagnostics/use-of-deprecated-aggregation/input.xml

```diff
@@ -0,0 +1,8 @@
+<mvc:View xmlns:mvc="sap.ui.core.mvc" xmlns:charts="sap.ca.ui.charts">
```

Comment:
why did we add a test case for deprecated aggregation **tag** in a PR implementing deprecated **attributes** validation?